content/en/docs/concepts/storage/volume-pvc-datasource.md

```diff
@@ -0,0 +1,61 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- thockin
+- msau42
+title: CSI Volume Cloning
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.15" state="alpha" >}} 
+This document describes the concept of cloning existing CSI Volumes in Kubernetes.  Familiarity with [Volumes](/docs/concepts/storage/volumes) is suggested.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Introduction
+
+The CSI Volume Cloning feature adds support for specifying existing PVCs in the `dataSource` field to indicate a user would like to Clone a Volume.
+
+A Clone is defined as a duplicate of an existing Kubernetes Volume that can be consumed as any standard Volume would be.  The only difference is that upon provisioning, rather than creating a "new" empty Volume, the back end device creates an exact duplicate of the specified Volume.
+
+The implementation of Cloning, from the perspective of the Kubernetes API is the same as "creating volumes from snapshots", it leverages the DataSource field in the PVC spec.
+
+Users need to be aware of the following when using this feature:
+
+* Cloning support (`VolumePVCDataSource`) is only available for CSI drivers.
+* Cloning support is only available for dynamic provisioner's
```

Comment:
Do you mean 'CSI volume cloning is only available for dynamically provisioned CSI volumes?'

Reply:
Right, the requirements are CSI and Dynamic, thus "dynamic CSI provisioner".